spec/controller/users_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET/index' do
```

Comment:
- [ ] Your user Spec file should include tests that ensure the correct view template is rendered. Example : `expect(response).to render_template(:show)`
- Kindly add this ✍🏻